docs/examples/tutorial/embedding/embedded.pyx

```diff
@@ -3,6 +3,7 @@
 # The following two lines are for test purposes only, please ignore them.
 # distutils: sources = embedded_main.c
 # tag: py3only
+# tag: no-cpp
```

Comment:
This seems an unrelated change that shouldn't be included in this PR.

Reply:
The intention is to exclude this test from C++ backend runs.
Running this test with the C++ backend fails: `.pyx` file is cythonized to C++, and the generated C++ header is included in a `.c` compilation unit, and finally the compiler rightfully rejects the `extern "C++"` declaration.

If we want to keep it in the C++ backend, then the proper solution would be to redefine the `__PYX_EXTERN_C` macro.